esp/bindings/http/platform/httpbinding.hpp

```diff
@@ -321,6 +334,33 @@ class esp_http_decl EspHttpBinding :
         return false;
     }
     ISecManager* querySecManager() {return m_secmgr.get(); }
+    IAuthMap* queryAuthMAP(){return m_authmap.get();}
+    const char* queryAuthMethod() {return m_authmethod.str(); }
+    void setProcessName(const char* name){ processName.set(name); }
+    const char* getProcessName(){ return processName.get(); }
+    void setDomainName(const char* name){ domainName.set(name ? name : "default"); }
+    const char* getDomainName(){ return domainName.get(); }
+    void setDomainSessionSDSPath(const char* path){ domainSessionSDSPath.set(path); }
+    const char* getDomainSessionSDSPath(){ return domainSessionSDSPath.str(); }
+    const char* getSessionIDCookieName() {return sessionIDCookieName.str(); }
+    AuthType getDomainAuthType(){ return domainAuthType; }
+    const char* getLoginURL() { return loginURL.get(); }
+    const char* getLogoutURL() { return logoutURL.get(); }
```

Comment:
minor: some inconsitency above between queryXX methods and getXX methods.
Normally the above 2 gets would be queryXX, as per queryAuthMethod()